server/controllers/admin.py

```diff
@@ -290,7 +302,7 @@ def autograde_backup(bid):
     return redirect(url_for('.grading', bid=bid))
 
 @admin.route("/versions/<name>", methods=['GET', 'POST'])
-@is_staff()
+@is_admin()
```

Comment:
There are definitely still use cases for `is_staff()` like the listing of all courses or the grading view. There are a few views in the admin interface that don't know/have a course attached - but also aren't admin actions.